TASK-50 Add files logic

KirSanich opened the pull request:
Создание каркаса сервиса внешних файлов, реализация основного функционала для работы с базой данных и s3 хранилищем.

Commit:
TASK-50 Add-files-logic

Создание каркаса сервиса внешних файлов, реализация основного функционала
для работы с базой данных и s3 хранилищем.

uncmmath commented:
А так ли нужны все эти сущности из core в external-files? у нас же модуль этот по сути занимается только работой с файлами

ещё немного не по теме:
вообще я так подумал, что при изменении сущностей будет немного гемморно их синхронизировать между проектами, если у нас будет сильная связность, мб имеет смысл вынести их в отдельный модуль и юзать из других модулей?

KirSanich commented:
По поводу сущностей, я полностью миграции поднимал, поэтому все сущности нужны были, пока не совсем понятно что надо или нет, там ведь связи есть, уберу сущность - пропадет связь и не скомпилируется.

По поводу модуля я предлагал в начале проекта, но все проголосовали за обычное копирования, то что модуль удобнее знаю, на одном из проектов знакомый из команды подключал на мавене жарник, хотя и там тоже мозготравли с ним много.

Commit:
TASK-50 Add-files-logic

Добавление парсинга файла кандидатов от hr(альфа-версия), требует доработки и рефакторинга.
